src/Nethermind/Nethermind.Synchronization/SyncServer.cs

```diff
@@ -326,13 +326,13 @@ private void LogBlockAuthorNicely(Block block, ISyncPeer syncPeer)
             if (block.Author is not null)
             {
                 sb.Append(" sealer ");
-                if (KnownAddresses.GoerliValidators.ContainsKey(block.Author))
+                if (KnownAddresses.GoerliValidators.TryGetValue(block.Author, out string value))
                 {
-                    sb.Append(KnownAddresses.GoerliValidators[block.Author]);
+                    sb.Append(value);
                 }
-                else if (KnownAddresses.RinkebyValidators.ContainsKey(block.Author))
+                else if (KnownAddresses.RinkebyValidators.TryGetValue(block.Author, out value))
                 {
-                    sb.Append(KnownAddresses.GoerliValidators[block.Author]);
```

Comment:
This one is also a bugfix. Nice! As a relief, it was with Rinkeby that shuts down mid-this year.